client/src/modules/fill_form/modals/fill_form.modals.html

```diff
@@ -0,0 +1,250 @@
+<form name="FillFormForm" bh-submit="FillFormModalCtrl.submit(FillFormForm)" novalidate>
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static" translate>TREE.DATA_COLLECTION</li>
+
+      <li class="title">
+        <span translate>TREE.FORMS_CONFIGURATION</span>
+      </li>
+    </ol>
+  </div>
+
+  <div style="background-color: {{ FillFormModalCtrl.dataCollector.color }}; border-radius : 25px ; text-align: center; text-decoration-color: #ffffff; text-shadow: rgb(255, 255, 255) 5px 3px 8px;">
+    <h3><strong>
+      <span translate>
+        {{ FillFormModalCtrl.dataCollector.label }}
+      </span>
+    </strong></h3>
+    <h4><strong ng-if="FillFormModalCtrl.patient.display_name"> {{ FillFormModalCtrl.patient.display_name }} / {{ FillFormModalCtrl.patient.reference}} ({{ FillFormModalCtrl.patient.sex }}) </strong></h4>
+  </div>
+
+  <div class="modal-body" popover-append-to-body="true" style="overflow-y: scroll; min-height:300px; max-height:600px; ">    
+    <div ng-repeat="formItem in FillFormModalCtrl.formItems">
+
+      <div ng-if="!formItem.constraint || $eval(formItem.constraintCheck)">
+        <!-- Type text -->
+        <div ng-if="formItem.typeForm === 'text'" class="form-group" ng-class="{ 'has-error' : FillFormForm.$submitted && FillFormForm.{{ formItem.name }}.$invalid }">
+          <label class="control-label"> {{ formItem.label }} </label>
+          <span style="color: #cecece"><em> {{ formItem.hint }} </em></span>
+          <div>
+            <span ng-if="formItem.required">
+              <input name="{{ formItem.name }}" ng-model="FillFormModalCtrl.form[formItem.name]" autocomplete="off" ng-maxlength="90" class="form-control" required>
+            </span>
+            <span ng-if="!formItem.required">
```

Comment:
Why not use [`ngRequired`](https://docs.angularjs.org/api/ng/directive/ngRequired)?